src/Link/Link.tsx

```diff
@@ -22,11 +22,23 @@ const hoverColor = system({
 
 const StyledLink = styled.a<StyledLinkProps>`
   color: ${props => (props.muted ? get('colors.fg.muted')(props) : get('colors.accent.fg')(props))};
-  text-decoration: ${props => (props.underline ? 'underline' : 'var(--prefers-link-underlines, underline)')};
+  text-decoration: ${props => {
+    if (props.underline === undefined) {
+      return 'var(--prefers-link-underlines, underline)'
```

Comment:
Assuming `BaseStyles` is an optional wrapper... (correct me if it's not!)

Right now, in `main` we set the default to `none`.

```ts
text-decoration: props.underline ? 'underline' : 'none'
```

Instead of changing the default, should we keep that default and let folks change the default by _switching-on_ `--prefers-link-underlines`? 

```ts
text-decoration: props.underline ? 'underline' : 'var(--prefers-link-underlines, none)'
```